testsuite/tests/service_mesh/auth/wasm/test_mapping_rules_sync.py

```diff
@@ -0,0 +1,51 @@
+"""
+Tests wasm mapping rules special handling.
+"""
+import backoff
+import pytest
+from testsuite.capabilities import Capability
+from testsuite import rawobj
+from testsuite.utils import blame
+
+
+pytestmark = pytest.mark.required_capabilities(Capability.SERVICE_MESH)
+
+
+@pytest.fixture(scope="session")
+def set_mapping_rules(request):
+    """Deletes implicit '/' rule and sets new for path '/anything/something'"""
+    def _set_mapping_rules(service):
+        proxy = service.proxy.list()
+        proxy.mapping_rules.delete(proxy.mapping_rules.list()[0]["id"])  # delete implicit "/" rule
+        metric = service.metrics.create(rawobj.Metric(blame(request, "metric")))
+        proxy.mapping_rules.create(rawobj.Mapping(metric, pattern="/anything/something"))
+        proxy.deploy()
+    return _set_mapping_rules
+
+
+@pytest.fixture(scope="session")
+def backoff_request():
```

Comment:
this does not have to be a fixture or bit better to say this should not be the fixture

Reply:
Would it be good to move this function to testsuite.utils maybe?